.github/workflows/jacoco_check.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       - name: Add coverage to PR
         if: steps.jacocorun.outcome == 'success'
         id: jacoco-agent
-        uses: madrapps/jacoco-report@${{ env.jaCocoReportVersion }}
+        uses: madrapps/jacoco-report@v1.6.1
```

Comment:
Still hardcoded.

Reply:
I filled it as a variable but failed to find a way to get through. The received error is that `env` cannot be recognized.